data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+# Copyright (c) 2020 by Fernando S. Sáez
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "The Box 1"
+	landing
+	name "The Sheragi bunker"
+	description "Visit Albert Foster on Zug to see what he discovered."
+	source
+		near "Zubeneschamali" 1 100
+	destination Zug
+	
+	to offer
+		has "event: rim archaeology results"
+		has "First Contact: Unfettered: declined"
+		has "main plot completed"
+		random < 4
+	
+	on offer
+		conversation
+			`You receive a message from Albert Foster, the archaeologist who you helped uncover the mysterious Sheragi ruins on Zug. He seems to be very excited.`
+			`	"<first>! It's been a while, huh? Remember the Sheragi dig site? I've just found something incredible here. Please, come visit me on Zug. I don't think I can trust anyone else with this."`
+				accept
+
+mission "The Box 2"
+	landing
+	name "The mysterious box"
```

Comment:
I don't think that mission names should be lowercase, but there might be precedent for that that I can't recall. If there is, let me know, and if there isn't, I would recommend capitalizing all of the mission names.